config/overlays/ca-cert.yaml

```diff
@@ -0,0 +1,20 @@
+#@data/values
+#@ load("@ytt:data", "data")
+#@ load("@ytt:assert", "assert")
+
+#@ def find_cert():
```

Comment:
I know @cppforlife discouraged us from doing this. I wonder if we should just provide ca cert as a regular value.

Reply:
ya that sounds less confusing anyways